generate/templates/templates/nodegit.cc

```diff
@@ -120,15 +121,44 @@ void OpenSSL_ThreadSetup() {
   CRYPTO_THREADID_set_callback(OpenSSL_IDCallback);
 }
 
+inline bool file_exists (const char* name) {
+  struct stat buffer;   
+  return (stat (name, &buffer) == 0); 
```

Comment:
do we have 
`std::filesystem::exists`
if we are using c++ 17?

will this works on windows?

Reply:
+1 on using a std lib

Reply:
I tried `std::filesystem::exists`, no success.  Btw this won't be enabled on windows, see that `ifdef` up a few lines